docs/layers/polygon-layer.md

```diff
@@ -172,11 +172,18 @@ Be aware that this prop will likely be changed in a future version of deck.gl.
 
 * default: `object => object.polygon`
 
-Like any deck.gl layer, the polygon accepts a data prop which is expected to
-be an iterable container of objects, and an accessor
-that extracts a polygon (simple or complex) from each object.
+Called on each object in the `data` stream to retrieve its corresponding polygon.
 
-This accessor returns the polygon corresponding to an object in the `data` stream. A polygon is expected to be an array of coordinates following either the GeoJSON [LineString](https://tools.ietf.org/html/rfc7946#section-3.1.4) ("simple polygon") or [Polygon](https://tools.ietf.org/html/rfc7946#section-3.1.6) (polygon with holes) specification. Each vertex can optionally supply a third component `z` which specifies the altitude of the vertex.
+A polygon can be one of the following formats:
```

Comment:
1. `line 65 - line 72` also describe polygon formats similarly.
2. Maybe add example dataset for all the four formats in the layer example